JSDemos/Demos/Toolbar/Overview/description.md

```diff
@@ -1 +1,35 @@
-The Toolbar is&nbsp;a&nbsp;UI component containing items that usually manage screen content. Those items can be&nbsp;plain text or&nbsp;widgets.
\ No newline at end of file
+The Toolbar contains items that manage the page content: other components and screen view. In this demo, the Toolbar manages the [List](/Documentation/ApiReference/UI_Components/dxList/).
+
+## Configure Toolbar Items
+
+You can store Toolbar items in the [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or specify the [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource). A Toolbar item may be plain text or a UI component. You should specify the [text](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#text) or the [widget](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#widget) property depending on the item. If the item is a UI component, declare its [options](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options).
```

Comment:
`You can store Toolbar items in the [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or specify the [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource).`

I want to make sure my understanding of this sentence is correct.
Is "store" the correct verb here, or do you mean that the toolbar items are _displayed_ from an array?
For example, do you mean the following?

`You can display Toolbar items from an [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or a [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource).`

If not, please clarify.

Reply:
Yes, "store" is not right, thank you for your suggestion, I will update it in the following commit.

Reply:
Thank you.